### PR TITLE
Move Monolog to Addons

### PR DESCRIPTION
**Pull request by nupplaphil**
Because Monolog is currently never used at any place, but it's still valuable for further logging implementations, I moved it into Addons and created a new Hook "logger_instance" for it.

### CODE REVIEW

File: doc/Addons.md [hunk 1]
```diff
@@ -919,6 +919,11 @@ Here is a complete list of all hook callbacks with file locations (as of 24-Sep-
     Hook::callAll('block', $hook_data);
     Hook::callAll('unblock', $hook_data);
 
+### src/Core/Logger/Factory.php
+
+	Hook::callAll('logger_instance', $data);
```
Comment (MrPetovan) [inline doc/Addons.md]: Wrong indentation in the md files.
```suggestion
    Hook::callAll('logger_instance', $data);
```

File: doc/de/Addons.md [hunk 2]
```diff
@@ -405,6 +405,10 @@ Eine komplette Liste aller Hook-Callbacks mit den zugehörigen Dateien (am 01-Ap
     Hook::callAll('block', $hook_data);
     Hook::callAll('unblock', $hook_data);
 
+### src/Core/Logger/Factory.php
+
+	Hook::callAll('logger_instance', $data);
```
Comment (MrPetovan) [inline doc/de/Addons.md]:
```suggestion
    Hook::callAll('logger_instance', $data);
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by nupplaphil** [comment]
@MrPetovan  - plz ignore the license_check .. bin/daemon.php check again
--> I'll fix it soon

**Review by MrPetovan — APPROVED** [review]

**Comment by nupplaphil** [comment]
sry @MrPetovan , there was a problem with Woodpecker, I squashed my changes and force-pushed it again to kick Woodpecker .. :-/ .. seems like Woodpecker was temporary overloaded

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Waiting for you to address the feedback in the addons PR.

**Pull request merged**